src/data/faq_de.json

```diff
@@ -282,7 +282,7 @@
                                 "anchor": "rat_multiple_rat",
                                 "active": false,
                                 "textblock": [
-                                    "Nein, es kann immer nur ein Schnelltest in der App registriert werden. Ein negativer Schnelltest wird für 48 Stunden in der App angezeigt. Nach Ablauf dieser Zeit können Sie einen neuen Test registrieren. Sie haben auch die Möglichkeit, den negativen Schnelltest und das Ergebnis zu entfernen, um anschließend einen neuen Schnelltest zu registrieren. Positive Schnelltests werden nicht mehr in der App angezeigt, sobald sie geteilt wurden.",
+                                    "Ja, seit Version 2.21 ist es möglich Schnelltests von Familienmitgliedern in der Corona-Warn-App zu verwalten. Pro Person gilt weiterhin: Ein negativer Schnelltest wird für 48 Stunden in der App angezeigt. Nach Ablauf dieser Zeit können Sie einen neuen Test registrieren. Sie haben auch die Möglichkeit, den negativen Schnelltest und das Ergebnis zu entfernen, um anschließend einen neuen Schnelltest zu registrieren. Positive Schnelltests werden nicht mehr in der App angezeigt, sobald sie geteilt wurden.",
                                     "PCR-Tests werden unabhängig von den Schnelltests behandelt. Das heißt, Sie können jeweils einen PCR-Test und einen Schnelltest zur gleichen Zeit in der App registriert haben."

```

Comment:
For each person?

Reply:
This is for each person and not in total. To make this more clear in the FAQ, I've removed the second paragraph and included all information into a list. See the images above.